src/ui/dialogs/muchelp.ts

```diff
@@ -0,0 +1,14 @@
+import Dialog from '../Dialog'
+
+const muchelpTemplate = require('../../../template/muchelp.hbs');
+
+export default function() {
+   let content = muchelpTemplate({});
```

Comment:
We should use camel case for the template file and maybe give it are more meaningful name.

Reply:
done... now mucHelpDialog...